io-engine/src/rebuild/rebuild_job.rs

```diff
@@ -292,6 +316,27 @@ impl<'n> RebuildJob<'n> {
             .collect()
     }
 
+    /// Create a RebuildRecord from RebuildJob
```

Comment:
In rust these are typically done using the From trait, so you'd impl From<RebuildJob> for RebuildRecord.

Reply:
ok, right will modify this.